sdk/scheduler/src/main/java/org/apache/mesos/specification/ContainerSpec.java

```diff
@@ -3,11 +3,16 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
+import java.util.Optional;
+
 /**
  * Spec for defining a Container.
  */
 @JsonDeserialize(as = DefaultContainerSpec.class)
 public interface ContainerSpec {
     @JsonProperty("image-name")
```

Comment:
Yeah, I followed the style with respect to every other member with the word "spec" in it, which all have underscores, but that's probably getting too specific -- we should probably standardize on one or the other.